packages/config/src/Config.ts

```diff
@@ -9,7 +9,7 @@
 *
 */
 
-import * as node_path from "path";
+import * as path from "path";
 import * as os from "os";
 import * as fs from "fs";
```

Comment:
I believe I changed this to be able to have a line that's is almost 150 characters long 😋 
I can revert it back (and maybe change all other packages we import from node to `node_<package>` (e.g. `node_os`, `node_fs`, ...)